engine/src/exec-interface-image.cpp

```diff
@@ -344,15 +363,32 @@ void MCImage::GetText(MCExecContext& ctxt, MCDataRef& r_text)
 		}
 
 		if (MCDataCreateWithBytes((const byte_t *)t_data, t_size, r_text))
-			return;
+        {
+            MCactiveimage = this;
```

Comment:
Do we need to set MCactiveimage here? As long as cloning the MCImage does not touch MCactiveimage, then there is no need.